crates/utils/re_log/src/lib.rs

```diff
@@ -73,9 +73,11 @@ const CRATES_AT_INFO_LEVEL: &[&str] = &[
     "h2",
     "hyper",
     "ureq",
-    // only let rustls run in debug mode: https://github.com/rerun-io/rerun/issues/3104
+    // only let rustls log in debug mode: https://github.com/rerun-io/rerun/issues/3104
     #[cfg(debug_assertions)]
     "rustls",
+    // winit 0.30.5 spams about `set_cursor_visible` calls. It's gone on winit master, so hopefully gone in next winit release.
+    "winit",
```

Comment:
add a `TODO`/issue to track that?

Reply:
We don't really have a system in place for tracking new releases… this is a very minor thing anyways (we rarely care about debug logs from winit)